plugins/reporters/fossid/src/main/kotlin/FossIdSnippetReporter.kt

```diff
@@ -38,9 +38,10 @@ import org.ossreviewtoolkit.reporter.ReporterInput
 class FossIdSnippetReporter(override val descriptor: PluginDescriptor = FossIdSnippetReporterFactory.descriptor) :
     Reporter by delegateReporter {
     companion object {
-        private const val TEMPLATE_NAME = "fossid_snippet"
-
-        private val delegateReporter = HtmlTemplateReporter(FossIdSnippetReporterFactory.descriptor)
+        private val delegateReporter = HtmlTemplateReporter(
+            FossIdSnippetReporterFactory.descriptor,
+            AsciiDocTemplateReporterConfig(templateIds = listOf("fossid_snippet"), templatePaths = null)
+        )
```

Comment:
Also here, how about moving non-public static field to the top-level?

Reply:
Do you mean `delegateReporter`? That was here before.

Reply:
Yes, I did mean that. Then let's keep it as-is.